src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
+        Random random = new Random();
```

Comment:
The same applies to Random class. You can create a single instance and reuse it instead of creating a new one every time the method is called. This can be a private class-level variable.